src/ntfs.rs

```diff
@@ -253,6 +255,61 @@ fn manage_feed_infos(collections: &mut Collections, path: &path::Path) {
     )
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct CommentLink {
+    object_id: String,
+    object_type: String,
+    comment_id: String,
+}
+
+fn insert_comment_link<T>(collection: &mut Collection<T>, comment_link: CommentLink)
+where
+    T: CommentLinks + Id<T>,
+{
+    let idx = match collection.get_idx(&comment_link.object_id) {
+        Some(idx) => idx,
+        None => {
+            error!(
+                "comment_links.txt: object_type={} object_id={} not found",
+                comment_link.object_type, comment_link.object_id
+            );
+            return;
+        }
+    };
+    collection
+        .index_mut(idx)
+        .comment_links_mut()
+        .push(comment_link.comment_id);
+}
+
+fn manage_comments(collections: &mut Collections, path: &path::Path) {
+    if path.join("comments.txt").exists() {
+        info!("Reading comments.txt");
+        collections.comments = make_collection(path, "comments.txt");
+
+        if let Ok(mut rdr) = csv::Reader::from_path(path.join("comment_links.txt")) {
+            info!("Reading comment_links.txt");
+            for comment_link in rdr.deserialize().map(Result::unwrap) {
+                let comment_link: CommentLink = comment_link;
+                match comment_link.object_type.as_str() {
+                    "stop_area" => insert_comment_link(&mut collections.stop_areas, comment_link),
+                    "stop_point" => insert_comment_link(&mut collections.stop_points, comment_link),
+                    "line" => insert_comment_link(&mut collections.lines, comment_link),
+                    "route" => insert_comment_link(&mut collections.routes, comment_link),
+                    "trip" => insert_comment_link(&mut collections.vehicle_journeys, comment_link),
+                    // TODO find something better
+                    // "stop_time" => {
+                    //     insert_comment_link(&mut collections.stop_times, comment_link)
+                    // }
+                    // TODO
+                    // "line_group" => insert_comment_link(&mut collections.line_groups, code),
```

Comment:
you may want to handle the case with a `warn!()` message to not crash if you have data with stop_time or line_group comment.